### PR TITLE
Start documentation on confidential

### PR DESCRIPTION
**Pull request by olethanh**

### CODE REVIEW

**Comment by hoh** [comment]
Should this documentation be available on docs.aleph.im ?

**Comment by olethanh** [comment]
that's internal documentation for developement we need more user friendly documentation for docs.aleph but let's wait till we have the aleph-client implementation

File: doc/confidential.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
+# Confidential computing
+
+Aleph vm offer to launch confidential VM with AMD SEV. This is also know as TEE, Trusted Execution Environment
```
Comment (hoh) [inline doc/confidential.md:3]:
```suggestion
Aleph-vm offers to launch confidential VM with AMD SEV. This is also know as TEE, Trusted Execution Environment.
```

File: doc/confidential.md [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+
+This is only supported for instance with the qemu hypervisor.
```
Comment (hoh) [inline doc/confidential.md:5]:
```suggestion
This is only supported for instances with the Qemu as the hypervisor.
```

File: doc/confidential.md [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+
+## Life cycle
+First, a user creates a VM message and send it with notify_allocate. This simply notifies the API of the new VM.
```
Comment (hoh) [inline doc/confidential.md:8]:
```suggestion
First, a user creates a VM message and sends it with notify_allocate. This notifies the orchestrator about the creation of the new VM.  
```

File: doc/confidential.md [hunk 4]
```diff
@@ -0,0 +1,257 @@
+# Confidential computing
+
+Aleph vm offer to launch confidential VM with AMD SEV. This is also know as TEE, Trusted Execution Environment
+
+This is only supported for instance with the qemu hypervisor.
+
+## Life cycle
+First, a user creates a VM message and send it with notify_allocate. This simply notifies the API of the new VM.  
+The user fetch the platform certificate, validate it's chain again AMD root certificate.
```
Comment (hoh) [inline doc/confidential.md:9]:
```suggestion
The user fetches the platform certificate, validates it's chain again AMD root certificate.
```

**Review by hoh — COMMENTED** [review]
I see lots of small grammatical errors here. Can you use an LLM or a spell and grammar checker on this document ?

File: doc/confidential.md [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+
+After initializing the VM you can check it's status with:
+`sudo systemctl status aleph-vm-controller@decadecadecadecadecadecadecadecadecadecadecadecadecadecadecadeca.service`
+
+and see the logs with
+` sudo journalctl -u aleph-vm-controller@decadecadecadecadecadecadecadecadecadecadecadecadecadecadecadeca.service`
+
+**Important**
+ 
+If you modify your base image between test you will need to delete the image file on disk (which is a delta of the base image)
```
Comment (hoh) [inline doc/confidential.md:184]:
```suggestion
If you modify your base image between tests, you will need to delete the image file on disk (which is a delta of the base image)
```

**Review by hoh — COMMENTED** [review]
I added details and hyperlinks to the documentation.

File: doc/confidential.md [hunk 6]
```diff
@@ -0,0 +1,257 @@
...
+    
+
+# CRN Side
+
+## Hardware requirement
+An AMD process supporting SEV, minimum Epyc 3
+
+## Requirements for the CRN
+* Support must be enabled in the computer BIOS
+* The kernel and platform must support SEV.
```
[the remark this answers is not in the document]
Comment (olethanh) [reply doc/confidential.md:40]: yes 
This  is actually what we are using for testing.

File: doc/confidential.md [hunk 7]
```diff
@@ -0,0 +1,257 @@
...
+```
+    
+
+# CRN Side
+
+## Hardware requirement
+An AMD process supporting SEV, minimum Epyc 3
+
+## Requirements for the CRN
+* Support must be enabled in the computer BIOS
```
[the remark this answers is not in the document]
Comment (olethanh) [reply doc/confidential.md:39]: not sure the name would vary by BIOS  / motherboard manufacturer and it's Alie who enabled it on our server. But I will try to provide some links

File: doc/confidential.md [hunk 8]
```diff
@@ -0,0 +1,257 @@
...
+
+See AMD DOC for more info on enabling SEV for your system
+https://www.amd.com/fr/developer/sev.html
+
+
+## Enabling the confidential computing feature in aleph-vm
+
+Set the following environment variables to launch aleph-vm:
+```
+ALEPH_VM_ENABLE_QEMU_SUPPORT=1
```
[the remark this answers is not in the document]
Comment (olethanh) [reply doc/confidential.md:58]: it is not at the moment.

**Review by nesitor — APPROVED** [review]

**Pull request merged**